src/NuGet/Microsoft.CodeAnalysis.BuildClient.Package/Microsoft.CodeAnalysis.BuildClient.targets

```diff
@@ -0,0 +1,13 @@
+<Project>
+  <ItemGroup>
+    <Compile Include="$(MSBuildThisFileDirectory)..\..\Compilers\Core\CommandLine\BuildProtocol.cs" />
+    <Compile Include="$(MSBuildThisFileDirectory)..\..\Compilers\Core\CommandLine\CompilerServerLogger.cs" />
+    <Compile Include="$(MSBuildThisFileDirectory)..\..\Compilers\Core\CommandLine\NativeMethods.cs" />
+    <Compile Include="$(MSBuildThisFileDirectory)..\..\Compilers\Core\Portable\CommitHashAttribute.cs" />
+    <Compile Include="$(MSBuildThisFileDirectory)..\..\Compilers\Core\Portable\InternalUtilities\Debug.cs" />
+    <Compile Include="$(MSBuildThisFileDirectory)..\..\Compilers\Core\Portable\InternalUtilities\PlatformInformation.cs" />
+    <Compile Include="$(MSBuildThisFileDirectory)..\..\Compilers\Shared\BuildServerConnection.cs" />
+    <Compile Include="$(MSBuildThisFileDirectory)..\..\Compilers\Shared\NamedPipeUtil.cs" />
+    <Compile Include="$(MSBuildThisFileDirectory)..\..\Compilers\Shared\RuntimeHostInfo.cs" />
```

Comment:
It's always bugged me how this logic is spread out among so many files in different directories. Did you consider as part of this pulling it into a singe isolation directory?